src/app/donation-complete/donation-complete.component.html

```diff
@@ -14,115 +17,97 @@
-            <li class="c-receipt__item b-db">
-              <h4 class="c-receipt__title b-light b-grey b-rt-sm">Your donation</h4>
-              <p class="b-rt-0 b-bold b-m-0">{{ donation.donationAmount | exactCurrency:donation.currencyCode }}</p>
-            </li>
-            <li class="c-receipt__item b-db">
-              <h4 class="c-receipt__title b-light b-grey b-rt-sm" *ngIf="donation.matchedAmount > 0">Match funds</h4>
-              <p class="b-rt-0 b-bold b-m-0" *ngIf="donation.matchedAmount > 0">{{ donation.matchedAmount | exactCurrency:donation.currencyCode }}</p>
-            </li>
-            <li *ngIf="giftAidAmount > 0" class="c-receipt__item b-db">
-              <h4 class="c-receipt__title b-light b-grey b-rt-sm">Gift Aid</h4>
-              <p class="b-rt-0 b-bold b-m-0">{{ giftAidAmount | exactCurrency:donation.currencyCode }}</p>
-            </li>
-          </ul>
-          <div class="c-receipt__tip c-misc" *ngIf="donation.feeCoverAmount > 0 || donation.tipAmount > 0">
-            <p *ngIf="donation.feeCoverAmount > 0" class="c-tip__heading">Thank you for covering the platform fee of {{ donation.feeCoverAmount | exactCurrency:donation.currencyCode }}. This enables {{ donation.charityName }} to benefit from the full value of your donation.</p>
-            <p *ngIf="donation.tipAmount > 0" class="c-tip__heading">Thank you for leaving a tip of {{ donation.tipAmount | exactCurrency:donation.currencyCode }} towards the Big Give.</p>
-            <p class="c-tip__description b-light b-grey b-rt-sm">Total amount you have been charged</p>
-            <p class="b-rt-0 b-bold b-m-0">{{ cardChargedAmount | exactCurrency:donation.currencyCode }}</p>
-          </div>
-          <div class="c-misc">
-            <p>Boost your impact by sharing on social media</p>
-            <a href="https://www.facebook.com/sharer/sharer.php?u={{ shareUrl }}" target="_blank" rel="noopener"><img class="c-social-image" src="/assets/images/share-facebook.png" alt="Facebook" aria-label="Share on Facebook" width="50"></a>
-            <a href="https://twitter.com/share?url={{ shareUrl }}&text={{ prefilledText }}" target="_blank" rel="noopener"><img class="c-social-image" src="/assets/images/share-twitter.png" alt="Twitter" aria-label="Share on Twitter" width="50"></a>
-            <a href="https://www.linkedin.com/cws/share?url={{ shareUrl }}" target="_blank" rel="noopener"><img class="c-social-image" src="/assets/images/share-linkedin.png" alt="LinkedIn" aria-label="Share on LinkedIn" width="50"></a>
-            <a href="https://api.whatsapp.com/send?text={{ prefilledText }}%0A%0A{{ shareUrl }}" target="_blank" rel="noopener"><img id="whatsapp" class="c-social-image" src="/assets/images/share-whatsapp.png" alt="WhatsApp" aria-label="Share on Whatsapp" width="50"></a>
-          </div>
+    <div class="thank-you">
+      <h2>Thank you!</h2>
+
+      <p class="thank-you-text">Your generous donation has<br/>been made successfully</p>
```

Comment:
I know it's late in the process to bring this up, but late perhaps better than never:

Do we need the word "successfully" here? It's maybe a bit of a pet peace of mine in website copy, I feel like it doesn't add any information and betrays a lack of confidence in the system be successful as a matter of course.

Also since they are the ones that made the donation maybe we don't need to tell them that it was made - what if instead we said something like "Your generous donation has been received."

Maybe a question for Beth?

Reply:
@bdsl I think I'll leave it as is for now as that's what's included in the Figma designs, website copy document, and because we saw yesterday that it's actually quite common to use the word 'successfully' in _success_ messages.

Reply:
OK